models/models/participant.py

```diff
@@ -101,7 +102,7 @@ class Participant(SMBase):
 
     id: int
     project: ProjectId
-    external_id: str = None
+    external_ids: dict[str, str]
```

Comment:
Can we leave the regular external_id here too (including on the Sample), so we can backwards compatible support putting the default ID here (like we do for graphql schema)?